x-pack/solutions/observability/plugins/metrics_data_access/common/inventory_models/host/index.ts

```diff
@@ -19,7 +28,10 @@ export const host = createInventoryModel('host', {
       defaultMessage: 'Host',
     }
   ),
-  requiredModule: 'system',
+  requiredIntegration: {
+    beats: 'system',
```

Comment:
Nit: you can use the constants here
```suggestion
    beats: SYSTEM_INTEGRATION,
    otel: HOST_METRICS_RECEIVER_OTEL,
```